web/src/components/pages/AddNewPlugin.tsx

```diff
@@ -1,12 +1,14 @@
-// import { withAuthenticationRequired, AuthenticationRequiredPage } from "@marketplace/auth";
+import { AuthenticationRequiredPage } from "@marketplace/auth";
 import AddNewPluginOrg from "@marketplace/components/organisms/AddNewPlugin";
 
 export type Props = {};
 
 const AddNewPlugin: React.FC<Props> = () => {
-  // TODO: add AuthenticationRequiredPage
-  return <AddNewPluginOrg isLoggedIn={false} />;
+  return (
+    <AuthenticationRequiredPage>
+      <AddNewPluginOrg isLoggedIn={false} />
```

Comment:
Can you import isAuthenticated from @marketplace/auth and pass that into isLoggedIn?

Reply:
I got it!